### PR TITLE
Move revision history to appendix and add new items based on 2nd CR.

### PR DESCRIPTION
**Pull request by msporny**
This PR creates a new "Revision History" section and adds updates made during the second Candidate Recommendation phase.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/779.html" title="Last updated on Jul 17, 2021, 5:55 PM UTC (490815d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/779/b57167a...490815d.html" title="Last updated on Jul 17, 2021, 5:55 PM UTC (490815d)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6217,6 +6116,159 @@ <h2>Changing the DID controller</h2>
...
+toward common DID URL implementation patterns.
+      </li>
+      <li>
+Elaborate upon when DID Documents are understood to start existing.
+      </li>
+      <li>
+Conversion of PNG diagrams to SVG diagrams.
+      </li>
+      <li>
+Rearranged order of Appendices to improve readability.
```
Comment (TallTed) [inline index.html:6145]:
```suggestion
Convert PNG diagrams to SVG diagrams.
      </li>
      <li>
Rearrange order of Appendices to improve readability.
```

File: index.html [hunk 2]
```diff
@@ -6217,6 +6116,159 @@ <h2>Changing the DID controller</h2>
...
+and for `nextUpdate` and `nextVersionId`.
+      </li>
+      <li>
+Clarification that `equivalentId` and `canonicalId` are optional.
+      </li>
+      <li>
+Addition of a definitions for "amplification attack" and "cryptographic suite".
+      </li>
+      <li>
+Replacing the use of `publicKeyBase58` with `publicKeyMultibase`.
```
Comment (TallTed) [inline index.html:6195]:
```suggestion
Replacement of `publicKeyBase58` with `publicKeyMultibase`.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Minor tweaks to make verb tenses more consistent

**Review by dlongley — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -6217,6 +6116,159 @@ <h2>Changing the DID controller</h2>
...
+      <li>
+Rearrange order of Appendices to improve readability.
+      </li>
+      <li>
+Update the IANA guidance as a result of the IETF Media Type Maintenance
+Working Group efforts.
+      </li>
+      <li>
+Remove at risk feature <code>alsoKnownAs</code> due to insufficient
+implementation experience.
```
Comment (brentzundel) [inline index.html:6153]: flagging this line as one that will need to be removed should the `alsoKnownAs` implementers meet the deadline

**Review by brentzundel — APPROVED** [review]

File: index.html [hunk 4]
```diff
@@ -6217,6 +6116,159 @@ <h2>Changing the DID controller</h2>
...
+      </li>
```
Comment (msporny) [inline index.html:6154]:
```suggestion
```

File: index.html (hunk 3 above)
Comment (msporny) [reply index.html]: Fixed in 490815df0e5605bbdba9c246b878b35b3b0e1700.

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**